src/libstd/sys_common/backtrace.rs

```diff
@@ -107,47 +107,66 @@ fn filter_frames(frames: &[Frame],
         "_ZN3std3sys3imp9backtrace",
         "ZN3std3sys3imp9backtrace",
         "std::sys::imp::backtrace",
+
         "_ZN3std10sys_common9backtrace",
         "ZN3std10sys_common9backtrace",
         "std::sys_common::backtrace",
+
         "_ZN3std9panicking",
         "ZN3std9panicking",
         "std::panicking",
+
         "_ZN4core9panicking",
         "ZN4core9panicking",
         "core::panicking",
+
         "_ZN4core6result13unwrap_failed",
         "ZN4core6result13unwrap_failed",
         "core::result::unwrap_failed",
+
         "rust_begin_unwind",
         "_ZN4drop",
         "mingw_set_invalid_parameter_handler",
     ];
     static BAD_PREFIXES_BOTTOM: &'static [&'static str] = &[
+        "_ZN4core9panicking",
+        "ZN4core9panicking",
+        "core::panicking",
+
         "_ZN3std9panicking",
         "ZN3std9panicking",
         "std::panicking",
+
         "_ZN3std5panic",
         "ZN3std5panic",
         "std::panic",
-        "_ZN4core9panicking",
-        "ZN4core9panicking",
-        "core::panicking",
-        "_ZN3std2rt10lang_start",
-        "ZN3std2rt10lang_start",
-        "std::rt::lang_start",
-        "panic_unwind::__rust_maybe_catch_panic",
-        "__rust_maybe_catch_panic",
-        "_rust_maybe_catch_panic",
-        "__libc_start_main",
+
         "__rust_try",
-        "_start",
-        "main",
         "BaseThreadInitThunk",
         "RtlInitializeExceptionChain",
         "__scrt_common_main_seh",
         "_ZN4drop",
         "mingw_set_invalid_parameter_handler",
+
+        "_ZN4core3ops6FnOnce9call_once",
+        "ZN4core3ops6FnOnce9call_once",
+        "core::ops::FnOnce::call_once",
+
+        // tests
+        "_ZN91_$LT$std..panic..AssertUnwindSafe$LT$F$GT$$u20$as$u20$core..ops..FnOnce\
+         $LT$$LP$$RP$$GT$$GT$9call_once",
+        "ZN91_$LT$std..panic..AssertUnwindSafe$LT$F$GT$$u20$as$u20$core..ops..FnOnce\
+         $LT$$LP$$RP$$GT$$GT$9call_once",
+        "<std::panic::AssertUnwindSafe<F> as core::ops::FnOnce<()>>::call_once",
```

Comment:
I don't think that these are correct to have because this is just the beginning of a `catch_unwind`, right?